velox/functions/prestosql/JsonFunctions.h

```diff
@@ -46,4 +46,23 @@ struct JsonExtractScalarFunction {
   }
 };
 
+// json_array_length(json) -> bigint
+// Returns the array length of json (a string containing a JSON array)
+// Returns 0 if json is not an array.
```

Comment:
`json_array_length` should return `NULL` when the input json does not represent an array.

Also, let's add a description of this function in velox/docs/functions/json.rst and remove the comment here. Feel free to copy the description from Presto's documentation: https://prestodb.io/docs/current/functions/json.html?highlight=json_array_length#json_array_length.

Reply:
Thanks, modified accordingly